### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by jasontwong**
Why:
- We want a starting point for the booster kit.

This change addresses the need by:
- Doing some initial setup stuff.

### CODE REVIEW

**Comment by bbrock25** [comment]
👍

File: .flowconfig [hunk 1]
```diff
@@ -0,0 +1,97 @@
+[ignore]
+
+# We fork some components by platform.
+.*/*.web.js
+.*/*.android.js
+
+# Some modules have their own node_modules with overlap
+.*/node_modules/node-haste/.*
+
+# Ugh
```
Comment (samcdavid) [inline .flowconfig:10]: Ain't that the truth :smile:

Comment (jasontwong) [inline .flowconfig:10]: That's part of the react-native init. Kind of makes you wonder...

**Comment by samcdavid** [comment]
👍

**Pull request merged**